includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -893,6 +898,28 @@ public function link_session_to_customer( $session_id, $customer_id ) {
 		);
 	}
 
+	/**
+	 * Sends the contents of the "forterToken" cookie to the server.
+	 *
+	 * @param string $token Contents of the "forterToken" cookie, used to identify the current browsing session.
+	 *
+	 * @return array An array, containing a `success` flag.
+	 *
+	 * @throws API_Exception If an error occurs.
+	 */
+	public function send_forter_token( $token ) {
+		return $this->request(
+			[
+				'token'      => $token,
+				//phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+				'user_agent' => $_SERVER['HTTP_USER_AGENT'] ?? '',
```

Comment:
Wouldn't it be better to sanitize this input in case someone sends a malicious user-agent?

Reply:
I don't know what "malicious" would mean in this case. It will be passed directly to the server, and the server will pass it directly to Forter. It's not going to be stored in a database, it's not going to be presented to clients (no XSS potential). I feel like sanitizing the input here would be pointless.

Reply:
All right. I was concerned that this value would be written to the site's logs and possibly available on the dashboard, but if there's no risk with that we can leave it as is.